src/test/ui/issues/issue-45829/rename-extern.stderr

```diff
@@ -0,0 +1,16 @@
+error[E0259]: the name `issue_45829_a` is defined multiple times
+  --> $DIR/rename-extern.rs:15:1
+   |
+LL | extern crate issue_45829_a;
+   | --------------------------- previous import of the extern crate `issue_45829_a` here
+LL | extern crate issue_45829_b as issue_45829_a;
+   | ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
+   | |
+   | `issue_45829_a` reimported here
+   | You can use `as` to change the binding name of the import
```

Comment:
This seems to be incorrect output, the suggestion is not actually being shown. I believe this is because of the line of code you identified in the ticket.